contracts/Buffer.sol

```diff
@@ -121,7 +121,7 @@ library Buffer {
         }
 
         // Copy remaining bytes
-        uint mask = 256 ** (32 - len) - 1;
+        uint mask = 256 ** ((32 - len) - 1);
```

Comment:
This is wrong - it should be left as-is, or made `(256 ** (32 - len)) - 1`.

Reply:
If it's generating an error as-is you can make it unchecked.

Reply:
Both `256 ** ((32 - len) - 1);`(original) and `(256 ** (32 - len)) - 1`(your suggestion) will make all tests fail. What do you mean by `make it unchecked`?

Reply:
Wrapped with `unchecked` block